tests/playbooks/hsm/hsm_adoption.yml

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
I don't think this is the way the adoption code is invoked. see test-minimal to see how the code is invoked.